test/com/dataengi/crm/identities/daos/UsersDAOSpec.scala

```diff
@@ -54,12 +91,128 @@ class UsersDAOSpec
       user.loginInfo === TestLoginInfo
     }
 
+    "find user by companyId negative" in {
+      val NewCompanyName: String          = "NEW_COMPANY" + Gen.alphaStr.sample.get
+      val newCompanyResult: XorType[Long] = companiesService.create(NewCompanyName).await()
+      val newCompany                      = companiesService.get(newCompanyResult.value).await().value
```

Comment:
Logic of creating company is duplicated in each test case.

Code:
```scala
val NewCompanyName: String          = "NEW_COMPANY" + Gen.alphaStr.sample.get
val newCompanyResult: XorType[Long] = companiesService.create(NewCompanyName).await()
val newCompany                      = companiesService.get(newCompanyResult.value).await().value
```
can be extracted to function
```scala
def createCompany(): Company = {
   val NewCompanyName: String          = "NEW_COMPANY" + Gen.alphaStr.sample.get
   val newCompanyResult: XorType[Long] = companiesService.create(NewCompanyName).await()
   val getCompanyResult                = companiesService.get(newCompanyResult.value).await().value

   getCompanyResult.isRight === true
 
   val company = getCompanyResult.value

   company.id must not be None
   company
}
```
Then in each case

```scala
"find user by companyId negative" in {
  val company   = createCompany()
  val companyId = newCompany.id.get
...
}
```